website/package.json

```diff
@@ -4,15 +4,16 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
-    "build": "docusaurus build",
+    "start": "yarn run graphql-docs && docusaurus start",
```

Comment:
From this, it seems that the graphql docs are regenerated every time you run `npm start` or `npm build`. It's possible that, when testing unrelated changes with `npm start`, the git diff would include changes arising from this graphql doc regeneration, which a developer would need to manually unstage before committing. Is there any way to avoid this?

I had a similar issue for the Node.JS docs which are also auto-generated and in that case, I added the auto-generated docs to the .gitignore file. It is not a perfect solution though.

Reply:
Yep, you are right, it's not necessary to commit build artifacts, so we should put the `static/graphql` folder in .gitignore.
Polluting both the `start` and `build` commands was a concern I had when I implemented it because of the reason you mention. 
The other alternative would be to modify the Netlify build command in `netlify.toml` so the docs are only generated at build time in Netlify, and if we need to test them locally, just run `graphql-docs` and `docusaurus run/build` separately.